### ISSUE TITLE
Focus outline gets cut off in nav bar

### ISSUE THREAD

**Issue by EmmaDawsonDev** [sano2019/portfolio_react#4]
<img width="194" alt="image" src="https://user-images.githubusercontent.com/57045550/194063155-4d144bb6-7613-42db-aa57-3a9db844773b.png">

The top and bottom of the focus indicator are not visible when tabbing through the nav bar.

**Comment by Pranav4399** [comment]
I can work on this incase no one has already started.

**Comment by Im-Kunal-13** [comment]
@EmmaDawsonDev Can you assign this issue to me?

**Comment by Pranav4399** [comment]
I believe my commit for this issue has already been merged. In that case, this issue can be closed. Please look into this.

**Comment by sano2019** [comment]
> I believe my commit for this issue has already been merged. In that case, this issue can be closed. Please look into this.

Hi, I haven't merged anything yet. I need some time to review as I am working, so might take a little bit. Probably not today, please be patient, I will make sure to do it when I have time.

**Comment by sano2019** [comment]
This has been fixed.
